app/api/graphkb.js

```diff
@@ -72,9 +72,13 @@ const graphkbReviewStatus = async (graphkbToken, graphkbEntries) => {
   const {uri} = CONFIG.get('graphkb');
 
   const filters = {
-    '@rid': graphkbEntries.map(({kbStatementId}) => {
-      return kbStatementId;
-    }),
+    '@rid': graphkbEntries
```

Comment:
Small thing, but can you replace the .filter and .map with a single .reduce. Should look something like this:


```suggestion
    '@rid': graphkbEntries
        .reduce((filtered, {kbStatementId}) => {
            if (kbStatementId.match(/^#\d+:\d+$/)) {
                filtered.push(kbStatementId);
            }
            return filtered;
        }, []);
```